Does not start

Does not seem to work on my windows machine

OS: Windows 10 1909
NPM version: 6.14.6
Node version: v12.18.2

Steps to reproduce:
Clone repo
npm i
npm start

Expected output:
webserver to start

Actual output:
```
13:06:27 - Starting compilation in watch mode...

13:06:28 - Found 0 errors. Watching for file changes. -- stays here for minutes, no output, http://localhost:8080/ is not available
```

Interestingly running tsc and webpack-dev-server manually with npx produces a bunch of errors and a chokidar error for swapfiles:

```
ERROR in ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
Module not found: Error: Can't resolve 'a' in 'MYDIRPATHREMOVED\AmazonIVS\node_modules\amazon-ivs-player\dist\assets'
 @ ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
 @ ./src/basic.ts

ERROR in ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
WebAssembly module is included in initial chunk.
This is not allowed, because WebAssembly download and compilation must happen asynchronous.
Add an async splitpoint (i. e. import()) somewhere between your entrypoint and the WebAssembly module:
* ./src/basic.ts --> ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
* ./src/videojs-tech.ts --> ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
* multi (webpack)-dev-server/client?http://localhost:8080 ./src/basic.ts --> ./src/basic.ts --> ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
* multi (webpack)-dev-server/client?http://localhost:8080 ./src/videojs-tech.ts --> ./src/videojs-tech.ts --> ./node_modules/amazon-ivs-player/dist/assets/amazon-ivs-wasmworker.min.wasm
i ｢wdm｣: Failed to compile.
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\pagefile.sys'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\swapfile.sys'
```

--- a/package.json
+++ b/package.json
@@ -6,8 +6,9 @@
   "main": "",
   "scripts": {
     "build": "webpack --config ./webpack.config.js",
-    "tsc-watch": "tsc-watch --project ./tsconfig.json --noEmit",
-    "start": "npm run tsc-watch & webpack-dev-server --config ./webpack.config.js --watch --compress"
+    "watch:tsc": "tsc-watch --project ./tsconfig.json --noEmit",
+    "watch:webpack": "webpack-dev-server --config ./webpack.config.js --watch --compress",
+    "start": "npm-run-all --parallel watch:*"
   },
   "author": {
     "name": "Amazon Web Services",
@@ -27,6 +28,7 @@
     "@types/video.js": "^7.3.8",
     "babel-loader": "^8.1.0",
     "file-loader": "^6.0.0",
+    "npm-run-all": "^4.1.5",
     "tsc-watch": "^4.2.3",
     "typescript": "^3.8.3",
     "webpack": "^4.43.0",

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -61,7 +61,7 @@ module.exports = {
                  * The webpack file-loader (https://webpack.js.org/loaders/file-loader/) accomplishes this.
                  * Rollup's plugin-url (https://github.com/rollup/plugins/tree/master/packages/url) also seems to do this, but has not been tested.
                  */
-                test: /\/amazon-ivs-player\/.*dist\/assets\//,
+                test: /[\/\\]amazon-ivs-player[\/\\].*dist[\/\\]assets[\/\\]/,
                 loader: 'file-loader',
                 type: 'javascript/auto',
                 options: {

